BinaryEncoding.md

```diff
@@ -786,9 +783,140 @@ for [future :unicorn:][future multiple tables] use and must be 0 in the MVP.
 | `f32.reinterpret/i32` | `0xbe` | | |
 | `f64.reinterpret/i64` | `0xbf` | | |
 
+## Thread operators ([described here](Semantics.md#thread-operators))
+
+| Name | Opcode | Immediate | Description |
+| ---- | ---- | ---- | ---- |
+| `is_lock_free` | `0xff00` | | |
+| `i32.wait` | `0xff01` | `memory immediate` | |
+| `i64.wait` | `0xff02` | `memory immediate` | |
+| `i32.wake` | `0xff03` | `memory immediate` | |
+| `i64.wake` | `0xff04` | `memory immediate` | |
```

Comment:
Do we intend wait / wake to have sequentially-consistent semantics, or acquire (wait) / release (wake) sematics?